templates/zerver/integrations.html

```diff
@@ -2062,6 +2070,85 @@
 
     <div id="trello" class="integration-instructions">
 
+      <p>
+        First, create the stream you'd like to use for Trello notifications,
+        and subscribe all interested parties to this stream. We recommend the
+        name <code>trello</code>.
+      </p>
+
+      <p>
+        Your webhook URL is:<br/>
+        <code>https://api.zulip.com/v1/external/trello?api_key=abcdefgh&stream=trello</code><br/>
+        where <code>api_key</code> is the API key of your Zulip bot,
+          and <code>stream</code> is the stream name you want the notifications sent to.
+      </p>
+
+      <p>
+        Before you'll be able to create a Webhook,
+        you need to follow steps below to get an <b>APPLICATION_KEY</b> and a <b>UserToken</b>.
+        Also we need to fetch the <b>idModel</b> of the board.
```

Comment:
I suggest:
`Before you create a webhook, you'll need to follow the steps below to get an <b>APPLICATION_KEY</b> and a <b>UserToken</b>, and to fetch the board's <b>idModel</b>.`